crates/biome_cli/src/commands/migrate.rs

```diff
@@ -6,12 +6,14 @@ use biome_service::configuration::{load_configuration, LoadedConfiguration};
 use biome_service::ConfigurationBasePath;
 use std::path::PathBuf;
 
+use super::MigrateSubCommand;
+
 /// Handler for the "check" command of the Biome CLI
 pub(crate) fn migrate(
     session: CliSession,
     cli_options: CliOptions,
     write: bool,
-    prettier: bool,
+    sub_command: Option<MigrateSubCommand>,
 ) -> Result<(), CliDiagnostic> {
     let base_path = match cli_options.config_path.as_ref() {
```

Comment:
There's an API we can use, and it is also what other commands use. Would be good if we can apply the small fix in this PR :)

https://github.com/biomejs/biome/blob/3cbb65c62191a9609695296e47e2e91073c7ab4d/crates/biome_cli/src/cli_options.rs#L85

Reply:
Thanks for the suggestion!